package/MDAnalysis/coordinates/base.py

```diff
@@ -1782,31 +1770,3 @@ def range_length(start, stop, step):
     else:
         # The range is empty.
         return 0
-
-
-class _Convertermeta(type):
-    # Auto register upon class creation
-    def __init__(cls, name, bases, classdict):
-        type.__init__(type, name, bases, classdict)
-        try:
-            fmt = asiterable(classdict['lib'])
-        except KeyError:
-            pass
-        else:
-            for f in fmt:
-                f = f.upper()
-                _CONVERTERS[f] = cls
-
-class ConverterBase(IOBase, metaclass=_Convertermeta):
```

Comment:
We can't just purely move this out of here without some kind of deprecation window.

Could you maybe subclass the ConverterBase here and add a warning in `__init_subclass__`?

Reply:
Unfortunately that leads to a circular reference. Instead I include the code in both `base.py` files but have a deprecation warning when the `ConverterBase` from `coordinates` is subclassed.

Reply:
This solution works for me and the comments make clear what to do. I also really don't have a more elegant solution.

Reply:
Ok that makes sense - we definitely need tests to check that each respective subclass does / doesn't raise the warning.